docs/sources/panels/transformations/types-options.md

```diff
@@ -204,21 +202,42 @@ This would result in a table like this:
 | 2020-07-07 11:34:20 | Server A | EU         | 1     |
 | 2020-07-07 11:34:20 | Server B | EU         | 2     |
 
-**Value field name**
+### Value field name
 
 If you selected Server as the **Value field name**, then you would get one field for every value of the Server label.
 
 | Time                | Datacenter | Server A | Server B |
 | ------------------- | ---------- | -------- | -------- |
 | 2020-07-07 11:34:20 | EU         | 1        | 2        |
 
-For this example, I manually defined labels in the Random Walk visualization of TestData DB.
+### Merging behavior
+
+The labels to fields transformer is internally two separate transformations. One that acts on single series (extracting labels to fields) followed by the [merge](#merge) transformation to join all the results into a single table. 
+
+To illustrate this, here is another example where you have two queries that return time series with non overallaping labels. 
+
+* Series 1: labels Server=ServerA
+* Series 2: labels Datacenter=EU
+
+This will first result in these two tables
+
+| Time                | Server  | Value | 
+| ------------------- | ------- | ----- | 
+| 2020-07-07 11:34:20 | ServerA | 10
+
+| Time                | Datacenter | Value | 
+| ------------------- | ---------- | ----- |
+| 2020-07-07 11:34:20 | EU         | 20
 
-{{< docs-imagebox img="/img/docs/transformations/labels-to-fields-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+After merge: 
 
-After I apply the transformation, my labels appear in the table as fields.
+| Time                | Server  | Value | Datacenter |
+| ------------------- | ------- | ----- | ---------- |
+| 2020-07-07 11:34:20 | ServerA | 10    |            |
+| 2020-07-07 11:34:20 |         | 20    | EU         |
 
-{{< docs-imagebox img="/img/docs/transformations/labels-to-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+There is currently no option to disable this merge step. The merge transform tries to join 
```

Comment:
Should this information be before the examples? Something like:

The merge transformation tries to join all matching fields. This merge step is required and cannot be turned off.

Reply:
Yes, good idea. Moved to before examples